tests/Unit/AddContactTest.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+use Inertia\Testing\AssertableInertia as Assert;
+
+class AddContactTest extends \Tests\TestCase
+{
+    use \Illuminate\Foundation\Testing\DatabaseMigrations;
+
+    public function test_add_contact_with_first_and_last_name()
```

Comment:
So for simplicity you can name this testCreateContact

Secondly this looks like a test that actually calls a endpoint thus it is a System test

Unit > Class tests for mocking expected results from individual methods
System > Call a endpoint or DB with a repository and Assert expectations
Acceptance > Using something like Behat / automated website tests to run through the product and assert expectations